documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -50,9 +50,6 @@ Example output showing columns important to this procedure:
 [source,shell,subs="+quotes,attributes"]
 ----
 NAME                                         RECLAIMPOLICY CLAIM
-pvc-5e9c5c7f-3317-11ea-a650-06e1eadd9a4c ... Retain ...    myproject/data-my-cluster-zookeeper-1
-pvc-5e9cc72d-3317-11ea-97b0-0aef8816c7ea ... Retain ...    myproject/data-my-cluster-zookeeper-0
-pvc-5ead43d1-3317-11ea-97b0-0aef8816c7ea ... Retain ...    myproject/data-my-cluster-zookeeper-2
```

Comment:
I think this should stay. While the Topic Operator does not use Zookeeper to store the data anymore, it is still used to store Kafka metadata and the volumes need to be recovered!

Reply:
Returned!